src/runtime_src/hip/core/event.cpp

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright (C) 2024 Advanced Micro Device, Inc. All rights reserved.
+
+#include "event.h"
+
+namespace xrt::core::hip {
+
+bool
+command::
+submit()
+{}
+
+bool
+command::
+wait()
+{}
+
+bool
+event::
+submit() override
```

Comment:
compiler not throwing errors for this override? i dont think, we can specify override outside class definition.

Reply:
Same for all below overrides too.